test/unittests/stack_test.cpp

```diff
@@ -211,6 +211,13 @@ TEST(operand_stack, small_with_locals)
     EXPECT_EQ(stack.local(2).i64, 0xc2);
     EXPECT_EQ(stack.local(3).i64, 0xc3);
     EXPECT_EQ(stack.local(4).i64, 0xc4);
+
+    EXPECT_EQ(stack.pop().i64, 0xff);
+    EXPECT_EQ(stack.local(0).i64, 0xc0);
+    EXPECT_EQ(stack.local(1).i64, 0xc1);
+    EXPECT_EQ(stack.local(2).i64, 0xc2);
+    EXPECT_EQ(stack.local(3).i64, 0xc3);
+    EXPECT_EQ(stack.local(4).i64, 0xc4);
 }
```

Comment:
Can you also check here locals are unchanged?

Reply:
These checks check that locals are unchanged.